packages/replay/src/util/prepareReplayEvent.ts

```diff
@@ -30,12 +30,12 @@ export async function prepareReplayEvent({
 
   // extract the SDK name because `client._prepareEvent` doesn't add it to the event
   const metadata = client.getSdkMetadata && client.getSdkMetadata();
-  const name = (metadata && metadata.sdk && metadata.sdk.name) || 'sentry.javascript.unknown';
+  const { name, version } = (metadata && metadata.sdk) || {};
 
   preparedEvent.sdk = {
     ...preparedEvent.sdk,
-    version: __SENTRY_REPLAY_VERSION__,
-    name,
+    name: name || 'sentry.javascript.unknown',
+    version: version || '0.0.0',
```

Comment:
Just to be sure, `version` will already have been replaced somewhere else here, right?

Reply:
Yes, `version` is set automatically during SDK initialization: https://github.com/getsentry/sentry-javascript/blob/6557844d3eb91a2361d578bbbe3fd54de9e1308e/packages/react/src/sdk.ts#L9-L16

The reason for providing the fallback here is that users can potentially delete it or use a custom or an old client that doesn't yet have `getSdkMetadata`.